src/styl/_base/_typography.styl

```
@@ -29,7 +29,7 @@ h3
 h4
 h5
 h6
-    _fontload($fontstack-headings)
+    $fontstack-base
```

🤔  Isn't it more like
```suggestion
    font-family $fontstack-base
```
?